test/integration/local/test_serving.py

```diff
@@ -71,6 +71,7 @@ def test_serve_cpu_model_on_gpu(test_loader, docker_image, sagemaker_local_sessi
         _assert_prediction_npy_json(predictor, test_loader, content_types.NPY, content_types.JSON)
 
 
+@pytest.mark.skip_gpu_py2
```

Comment:
suggestion: should we also skip only if it's for 1.4?

Reply:
Ahh, that's right. Will follow up with a change to modify that.